lib/rust/ensogl/core/src/display/shape/primitive/def/primitive.rs

```diff
@@ -332,11 +332,20 @@ define_sdf_shapes! {
 
     // === Triangle ===
 
+    /// Isosceles Triangle
+    /// This is an exact SDF. Growing it will result in a triangle with rounded corners.
```

Comment:
What do you mean by "This is an exact SDF"?

Reply:
By that I mean it's not an approximation that holds true only inside the shape. The SDF function calculates a true distance, which is correct also far away from the shape boundary. That allows grow/shrink to work as expected. I've made a comparsion in shadertoy to illustrate that.

https://www.shadertoy.com/view/msG3W1
|Exact SDF|Bound SDF|
|-|-|
|![image](https://user-images.githubusercontent.com/919491/224285071-901613ed-bd9c-41ce-9e92-dada5dee30a3.png)|![image](https://user-images.githubusercontent.com/919491/224285180-abdcda9b-d650-4e9b-9a40-18e05e41bc51.png)|

Reply:
Oh, I when reading the comment I did not know that the sentence "This is an exact SDF" is explained by the following sentence! I think it would be clear if you use comma instead of dot in the description then ...